docusaurus.config.js

```diff
@@ -259,10 +259,17 @@ module.exports = {
         editUrl: 'https://github.com/SAP/cloud-sdk/edit/main',
         routeBasePath: 'docs/js',
         sidebarPath: require.resolve('./sidebarsDocsJs.js'),
-        lastVersion: 'current',
+        lastVersion: 'v2',
         versions: {
           current: {
+            label: 'v3',
+            banner: 'unreleased',
+            badge: false
```

Comment:
```suggestion
            badge: false,
            path: 'v3'
```

The paths have a `next` instead of `v3` (e.g. /cloud-sdk/docs/js/next/overview)